.github/workflows/docker.yml

```
@@ -0,0 +1,54 @@
+name: Push Docker images to DockerHub and ECR
+
+on:
+  push:
+    branches: [main, master]
+
+jobs:
+  multiple-registries:
```

```suggestion
  multiple-registries:
```
Non-blocking comment: I think this name could be better.